src/bounded_vec.rs

```diff
@@ -309,6 +308,7 @@ impl<T, const L: usize, const U: usize> BoundedVec<T, L, U> {
 
     /// Returns the last and all the rest of the elements
     pub fn split_last(&self) -> (&T, &[T]) {
+        const { assert!(L != 0) }
```

Comment:
crazy. need to change my pr to this if that works. like if assert does create new type constraint, also being just const assert.